notebook.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Clox"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Environment Check"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Checking that the notebook is running on Google Colab or not."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "try:\n",
+    "  import google.colab\n",
+    "  !{sys.executable} -m pip -q -q install clox\n",
+    "except:\n",
+    "  pass"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clox.functions import run_clock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "clock_parameters = {\n",
+    "    \"timezone\": None,\n",
+    "    \"country\": None,\n",
+    "    \"v_shift\": 0,\n",
+    "    \"h_shift\": 0,\n",
+    "    \"face\": 1,\n",
+    "    \"no_blink\": False,\n",
+    "    \"vertical\": False,\n",
+    "    \"hide_date\": False,\n",
+    "    \"hide_timezone\": False,\n",
+    "    \"am_pm\": False,\n",
+    "    \"date_system\": \"GREGORIAN\",\n",
+    "    \"date_format\": \"FULL\",\n",
+    "    \"offset_local\": 0,\n",
+    "    \"offset_timezone\": 0\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run_clock(**clock_parameters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Calendar"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clox.functions import print_calendar"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "calendar_parameters = {\n",
+    "    \"mode\": \"MONTH\",\n",
+    "    \"timezone\": None,\n",
+    "    \"country\": None,\n",
+    "    \"v_shift\": 0,\n",
+    "    \"h_shift\": 0,\n",
+    "    \"date_system\": \"GREGORIAN\",\n",
```

Comment:
Sorry I forgot that: If possible use our Enums for these cases; if we don't have enums comment other options that they had for users to be able to work with the example without the need to refer to the document.

Reply:
Good point

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/73/commits/ce12b374ab96493af12f5a179e6d52bc9b24aae6